features/events/syslog/src/main/java/org/opennms/netmgt/syslogd/ConvertToEvent.java

```diff
@@ -73,383 +73,468 @@
  */
 public class ConvertToEvent {
 
-    private static final Logger LOG = LoggerFactory.getLogger(ConvertToEvent.class);
```

Comment:
Is there any way to add participants/watchers to these comments?  Since our India guys were the actual authors (and I just handled the merge/push/pull request) it would be nice to somehow loop them in.  I can forward everything on, just seems like there has to be a better way.

On Dec 3, 2015, at 10:37 AM, Seth Leger <notifications@github.com<mailto:notifications@github.com>> wrote:


In features/events/syslog/src/main/java/org/opennms/netmgt/syslogd/ConvertToEvent.java<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_OpenNMS_opennms_pull_484-23discussion-5Fr46575670&d=BQMCaQ&c=NRtzTzKNaCCmhN_9N2YJR-XrNU1huIgYP99yDsEzaJo&r=l29111uPPfXaIWrnj7fnlA&m=3Uo_38AnwyPA9Et79ouKs67YBfojgvyMONYuOtfcg90&s=gRrHd29o0Za-fRC7zYaYWAhaFOUXiIPr-z0JkAdyNI0&e=>:

> @@ -73,383 +73,468 @@
>   */
>  public class ConvertToEvent {
>
> -    private static final Logger LOG = LoggerFactory.getLogger(ConvertToEvent.class);


This diff is too large because the whitespace was reformatted and the text was rewrapped. Can you undo the formatting changes so that it's easier to see the changed lines of code?

—
Reply to this email directly or view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_OpenNMS_opennms_pull_484_files-23r46575670&d=BQMCaQ&c=NRtzTzKNaCCmhN_9N2YJR-XrNU1huIgYP99yDsEzaJo&r=l29111uPPfXaIWrnj7fnlA&m=3Uo_38AnwyPA9Et79ouKs67YBfojgvyMONYuOtfcg90&s=68SH5249byPaY1hbCJRZPH8M36PJ8Q1gmzDhot_LuWo&e=>.

Doug Saus | Software Architect, CTS InfraOSS | Cerner Corporation | dsaus@cerner.com<mailto:dsaus@cerner.com> | O: 816-201-1058 | www.cerner.com<http://www.cerner.com/>


CONFIDENTIALITY NOTICE This message and any included attachments are from Cerner Corporation and are intended only for the addressee. The information contained in this message is confidential and may constitute inside or non-public information under international, federal, or state securities laws. Unauthorized forwarding, printing, copying, distribution, or use of such information is strictly prohibited and may be unlawful. If you are not the addressee, please promptly delete this message and notify the sender of the delivery error by e-mail or you may call Cerner's corporate offices in Kansas City, Missouri, U.S.A at (+1) (816)221-1024.